l1-contracts/test/foundry/unit/concrete/state-transition/chain-deps/facets/Mailbox/TransferEthToSharedBridge.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {MailboxTest} from "./_Mailbox_Shared.t.sol";
+import {IL1SharedBridge} from "contracts/bridge/interfaces/IL1SharedBridge.sol";
+
+contract MailboxTransferEthToSharedBridge is MailboxTest {
+    address baseTokenBridge;
+
+    function setUp() public virtual {
+        prepare();
+
+        baseTokenBridge = makeAddr("bridge");
+        vm.deal(diamondProxy, 1 ether);
+        utilsFacet.util_setChainId(eraChainId);
+        utilsFacet.util_setBaseTokenBridge(baseTokenBridge);
+    }
+
+    modifier useBaseTokenBridge() {
+        vm.startPrank(baseTokenBridge);
+        _;
+        vm.stopPrank();
+    }
+
+    function test_success_transfer() public useBaseTokenBridge {
+        vm.mockCall(baseTokenBridge, abi.encodeWithSelector(IL1SharedBridge.receiveEth.selector, eraChainId), "");
+
+        mailboxFacet.transferEthToSharedBridge();
```

Comment:
Please check for ETH balance updates on both the contracts.

Reply:
Added asserts